app/jobs/disable_inactive_profiles_job.rb

```diff
@@ -0,0 +1,18 @@
+class DisableInactiveProfilesJob < ApplicationJob
+  queue_as :low
+
+  def perform
+    Jobseeker.includes(:jobseeker_profile)
+             .active
+             .where(last_sign_in_at: ..6.months.ago)
+             .find_each
+             .filter_map(&:jobseeker_profile)
+             .select(&:active?)
+             .each do |jsp|
+      jsp.assign_attributes(active: false)
+      jsp.save!(touch: false)
```

Comment:
```suggestion
        JobseekerProfile
      .includes(:jobseeker)
      .where(jobseeker: { account_closed_on: nil, last_sign_in_at: ..6.months.ago }, active: true)
      .update_all(active: false)
```

Reply:
prefer my version - using the 'active' scope rather than account closed on